packages/plugin-ext/package.json

```diff
@@ -29,6 +29,7 @@
     "getmac": "^1.4.6",
     "jsonc-parser": "^2.0.2",
     "lodash.clonedeep": "^4.5.0",
+    "ps-tree": "^1.2.0",
```

Comment:
`ps-tree` is already declared with version `1.1.0` in `plugin-dev`, if you use that, there is no `yarn.lock` file changes.

If you insist on using `1.2.0`, can you please add the `@types/ps-tree` as well, so there is no need to use `any`s and `no-any`s linter rules. After adding the type definition for `ps-tree` as a dependency, you can do ES6 imports: `import * as psTree from 'ps-tree';`.

You're OK leaving everything as is.

Reply:
Thanks for your suggestion, I will use 1.2.0 and add @types/ps-tree.